portal/src/main/webapp/WEB-INF/jsp/mutation_details.jsp

```diff
@@ -76,6 +78,23 @@
         var sampleArray = window.QuerySession.getSampleIds();
         OncoKB.setUrl('<%=oncokbUrl%>');
         var mutationProxy = DataProxyFactory.getDefaultMutationDataProxy();
+        var annotationCol = null;
+
+        if(OncoKB.getAccess()) {
+            var oncokbInstanceManager = new OncoKB.addInstanceManager();
```

Comment:
@zhx828 Should we also move the initialization of OncoKB instance manager into the AnnotatorColumn class as well? Are we using OncoKB anywhere else?